tests/testthat/test-Source_datasets.R

```diff
@@ -190,12 +190,12 @@ test_that("SMSCG Meso column names have not changed", {
 })
 
 test_that("20mm Meso column names have not changed", {
-  expect_setequal(names_20mm_Meso, c('SampleDate', 'Survey', 'Station', 'TowNum', 'TowTime', 'Temp', 'TopEC', 'BottomEC', 'Secchi',
-                                     'Turbidity', 'Tide', 'BottomDepth', 'Duration', 'MeterCheck', 'Volume', 'Dilution',
+  expect_setequal(names_20mm_Meso, c('SampleDate', 'Year', 'Survey', 'Station', 'TowNum', 'TowTime', 'Temp', 'TopEC', 'BottomEC', 'Secchi',
+                                     'NTU','FNU', 'Tide', 'BottomDepth', 'Duration', 'MeterCheck', 'Volume', 'Dilution',
                                      'Acanthocyclops', 'Acanthomysis aspera', 'Acartia copepodid', 'Acartia spp_', 'Acartiella',
-                                     'Acartiella copepodid', 'Annelid worms', 'Asplanchna', 'barnacle nauplii', 'Bosmina',
```

Comment:
Did you reflect these changes to capitalization of taxa names in the crosswalk?

Reply:
Yup!